### PR TITLE
Content text

### PR DESCRIPTION
**Pull request by petros-chantz**

### CODE REVIEW

**Comment by petros-chantz** [comment]
Does it make sense now?

File: src/pages/Projects.js [hunk 1]
```diff
@@ -38,11 +38,7 @@ export default function Projects(props) {
   return (
     <PaddingProjects id="projects" gap="50px">
       <Container>
-        <Quotes
-          quotesText="Don’t tell me what the interaction is, just show it to me! So without further ado, here is a broad 
-        variety of projects I’ve worked on in the past 4 years spanning across UX Research,
-        UI Design and Frontend Development. Enjoy!"
-        />
+        <Quotes quotesText="So without further ado..." />
```
Comment (rania-chantz) [inline src/pages/Projects.js:41]: Auto den einai quote diko sou, auto einai genikh ekfrash pou uparxei. Kalutera auto "Don’t tell me what the interaction is, just show it to me!"

File: src/pages/Experience.js [hunk 2]
```diff
@@ -37,8 +37,8 @@ export default function Experience() {
       <Container padding="0px 50px 0px 0px">
         <Quotes
           quotesText="How are all these skills useful to you? Over the past 4+ years I have
```
Comment (rania-chantz) [inline src/pages/Experience.js:39]: H erwthsh den apantietai apo thn apanthsh. Epishs den einai quote, einai aplo keimeno. Mporoume na to kanoume apla keimeno?

**Comment by petros-chantz** [comment]
**NOT anymore relevant** branch merge

**Pull request closed without merging**